dev/puppet-server.conf.sample

```diff
@@ -115,3 +115,8 @@ authorization: {
         }
     ]
 }
+
+# general puppetserver settings
+puppetserver: {
+    environment-class-cache-enabled: true
+}
```

Comment:
We actually talked through this and chose this section *because* of UX :)  The thinking was:

"Why the hell do I need to know anything about jruby in order to tell you whether I want to cache environments or not?  What does JRuby have to do with caching or environments?"

Also, we're thinking that we may need to bust up that jruby section in some way before long, because of upcoming refactors to make the jruby service shareable for non-puppet use cases...